src/styles/calendar.scss

```
@@ -0,0 +1,60 @@
+@import "./variables.scss";
+
+.calendar {
+  padding: $calendar-padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  max-width: $calendar-widht;
+
+  @each $num, $day in $week_days {
+    &--start-day_#{$day} &__cell:first-of-type {
```

don't increase specificity if it's unnecessary, and it'd be better to use :first-child
```suggestion
    &--start-day_#{$day} :first-child {
```